Create Bruce_Wasson_Southwest_Systems.md

brucewasson99 opened the pull request:
For signing the neveragain.tech pledge.

ibash commented:
Hi Bruce, thank you for signing the pledge. We're trying to make sure that everyone who signs is who they say they are, and your name isn't associated with your github account.

Would you mind either putting a link to your github account from your linked in profile or another public website associated with your name, or follow the email instructions at https://github.com/neveragaindottech/neveragaindottech.github.io/blob/master/README.md

If you do create a link, please add a comment in the pull request letting us know where to find it and we'll take a look.

Thanks!

brucewasson99 commented on _signatures/Bruce_Wasson_Southwest_Systems.md:
```diff
@@ -0,0 +1,7 @@
+  occupation_title: "SVP"
+  affiliation: "Southwest Systems"
+  github: "brucewasson99"
+---
```
github account link: "Please see Websites in my LinkedIn Profile"

brucewasson99 commented:
Hi Islam,

I added a link to my GitHub account on my LinkedIn profile and added the
requested comment to the pull request.

Please let me know if there is anything else I can do for you. Thank you
very much for your time on this.

On Thu, Dec 15, 2016, 10:57 PM Islam Sharabash <notifications@github.com>
wrote:

> Hi Bruce, thank you for signing the pledge. We're trying to make sure that
> everyone who signs is who they say they are, and your name isn't associated
> with your github account.
>
> Would you mind either putting a link to your github account from your
> linked in profile or another public website associated with your name, or
> follow the email instructions at
> https://github.com/neveragaindottech/neveragaindottech.github.io/blob/master/README.md
>
> If you do create a link, please add a comment in the pull request letting
> us know where to find it and we'll take a look.
>
> Thanks!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/neveragaindottech/neveragaindottech.github.io/pull/1353#issuecomment-267530199>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AXdOJv3azkJvWkErcQONlHsbJLaH3P1Fks5rIjZpgaJpZM4LOocv>
> .
>
-- 
Best regards,

*Bruce Wasson*
Senior Vice President
Strategy Consultant
Southwest Systems Solutions
https://www.linkedin.com/in/brucewasson
M: 626.824.3695

lizthegrey commented:
Verified, thanks.

brucewasson99 commented:
Thank you Liz! I haven't been this happy to be on a list in a very long
time!

On Sat, Dec 17, 2016, 2:58 PM Liz Fong-Jones <notifications@github.com>
wrote:

> Merged #1353
> <https://github.com/neveragaindottech/neveragaindottech.github.io/pull/1353>
> .
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/neveragaindottech/neveragaindottech.github.io/pull/1353#event-897715855>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AXdOJl05j9Z9sZb50GiiKBBTZ1YX0Sidks5rJGkxgaJpZM4LOocv>
> .
>
-- 
Best regards,

*Bruce Wasson*
Senior Vice President
Strategy Consultant
Southwest Systems Solutions
https://www.linkedin.com/in/brucewasson
M: 626.824.3695
